static/scss/search.scss

```diff
@@ -82,6 +82,10 @@
   }
 
   .active-search-filters {
+    border-bottom: 1px solid black;
```

Comment:
in the mockup I think this is `1px solid #979797`

Reply:
(got that from https://projects.invisionapp.com/d/main/#/console/19832841/417550151/inspect)

Reply:
Good catch! I changed it to #8a8b8c to match the color of the document count divs. It definitely looks better than black.
<img width="456" alt="Screen Shot 2020-06-03 at 10 00 40 AM" src="https://user-images.githubusercontent.com/1934992/83646150-4712cc80-a581-11ea-96cb-5357aba556ca.png">